docs/source/art_inference_wrapper.rst

```diff
@@ -0,0 +1,23 @@
+ART Inference Attacks
+=====================
+
+.. currentmodule:: pepr.privacy.art_inference_wrapper
+
+Base wrapper class
+******************
+.. autoclass:: BaseMembershipInferenceAttack
```

Comment:
Why do we want to include the base wrapper class in the documentation? :)

Reply:
I thought it would be nice because it includes documentation about what can be expected in `attack_results` for example.

Reply:
In 809befa [here](https://github.com/hallojs/ml-pepr/pull/33/commits/809befa1ad7e191d8243c9cdc00439aaf4aa01e3#diff-3c90eda2c3843cef3bee4fd51a8d6cab570446f82e9b1d0842b1c7ec8da12f3aR8-R10) I added a short description, that the base class does not implement any attack to avoid misunderstanding.